lib/elixir/src/elixir_erl_try.erl

```diff
@@ -65,15 +65,17 @@ normalize_rescue(Meta, Var, Pattern, Expr, ErlangAliases) ->
   prepend_to_block(Meta, {'=', Meta, [Pattern, Normalized]}, Expr).
 
 dynamic_normalize(Meta, Var, [H | T]) ->
+  Generated = ?generated(Meta),
```

Comment:
I would go ahead and apply this to all of the `Meta`s in this function, no?

Reply:
I thought we usually apply the minimum, but it's done

Reply:
Since this whole thing is generated, I think it makes sense to mark it all. :)